config/schema/field_definitions.json

```diff
@@ -345,6 +345,38 @@
     "type": "float"
   },
 
+  "fault_type": {
+    "type": "identifiers"
+  },
+
+  "item_type": {
```

Comment:
`item_type` and `item_model` are pretty broad terms. Is it worth finding a more specific term? /cc @rboulton